config/local-development/vault-values.yaml

```diff
@@ -5,9 +5,24 @@ global:
 ui:
   enabled: true
 
+injector:
+  image:
+    tag: "1.1.0"
+    repository: docker.io/hashicorp/vault-k8s
+  agentImage:
+    repository: "docker.io/hashicorp/vault"
+    tag: "1.12.2"    
+
+csi:
+  image:
+    repository: "docker.io/hashicorp/vault-csi-provider"
+    tag: "1.2.1"
+    pullPolicy: IfNotPresent     
+
 server:
   image:
-    tag: "1.12.0"
+    tag: "1.12.2"
+    repository: docker.io/hashicorp/vault
```

Comment:
@raffaelespazzoli any reason for using the docker image vs https://github.com/hashicorp/vault-helm/blob/main/values.openshift.yaml#L17-L18

Reply:
actually, I'd do the change and recommit